core-features/napi-preview-feature/__snapshots__/test.ts.snap

```diff
@@ -0,0 +1,324 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`N-API Off 1`] = `
+Object {
+  "delete": Object {
+    "posts": Object {
+      "count": 1,
+    },
+    "users": Object {
+      "count": 1,
+    },
+  },
+  "post.findUnique": Object {
+    "id": 1,
+    "title": "Test",
+    "userId": 1,
+  },
+  "user.create": Object {
+    "email": "test@example.com",
+    "id": 1,
+    "name": "Test",
+    "posts": Array [
+      Object {
+        "id": 1,
+        "title": "Test",
+        "userId": 1,
+      },
+    ],
+  },
+  "users.findMany": Array [],
+}
+`;
+
+exports[`N-API Off 2`] = `
+"prisma               : placeholder
+@prisma/client       : placeholder
+Current platform     : placeholder
+Query Engine         : placeholder
+Migration Engine     : placeholder
+Introspection Engine : placeholder
+Format Binary        : placeholder
+Default Engines Hash : placeholder
+Studio               : placeholder"
+`;
+
+exports[`N-API Off: ./node_modules/.prisma/client 1`] = `
+Array [
+  "index-browser.js",
+  "index.d.ts",
+  "index.js",
+  "package.json",
+  "query-engine-debian-openssl-1.1.x",
```

Comment:
Simple way to test this on other platforms? Can we somehow have snapshots that depend on the OS this is running on? (We have some OS tests... maybe those help here somehow to figure this out.)

Reply:
I think the easiest way to do this would be to generate an object containing the expected files and switch the binary depending on the platform. We would also have to account for binary targets as these would add additional binaries.

Reply:
That sounds like a plan. Can you reflect that in an issue maybe? Or at least in the Napi Notion document list?